data/data-pipeline/data_pipeline/etl/sources/census_decennial/etl.py

```diff
@@ -147,6 +148,65 @@ def __init__(self):
             field_names.CENSUS_DECENNIAL_UNEMPLOYMENT_FIELD_2009
         )
 
+        # Race/Ethnicity fields
+        self.TOTAL_RACE_POPULATION_FIELD = "PCT086001"  # Total
+        self.ASIAN_FIELD = "PCT086002"  # Total!!Asian
+        self.BLACK_OR_AA_FIELD = "PCT086003"  # Total!!Black or African American
```

Comment:
This is a super nitty nit but everywhere else you just say `BLACK_FIELD` instead of `BLACK_OR_AA...` so maybe just drop the `OR_AA` for consistency?

Reply:
Good call --- when I started I was following the names in the census, then started using our other names for the ACS, and didn't go back and fix my original names. Will do so now, thank you!

Reply:
Fixed in b741001be6e1918273ddbc77f4431d507011d5df